cmd/run/message_input.go

```diff
@@ -41,17 +48,16 @@ func newMessageInput() *MessageInput {
 
 func (mi *MessageInput) reset() {
 	mi.SetTitle("")
-	mi.SetText("")
+	mi.SetText("", true)
 }
 
 func (mi *MessageInput) onInputCapture(event *tcell.EventKey) *tcell.EventKey {
 	switch event.Name() {
 	case config.Current.Keys.MessageInput.Send:
 		mi.sendAction()
 		return nil
-	case config.Current.Keys.MessageInput.Paste:
-		mi.pasteAction()
-		return nil
+	case "Alt+Enter":
```

Comment:
The "Alt+Enter" key event is redirected to the event queue as `tcell.KeyEnter` because `*tview.TextArea` handles`tcell.KeyEnter` in its `InputCapture` internally to insert a newline.; however, `tcell.KeyEnter` is handled in Discordo to send a new message by default.